packages/Upphandling/components/Issues.js

```diff
@@ -83,7 +83,7 @@ const styles = StyleService.create({
     borderRadius: 16,
   },
   body: {
-    overflow: '',
+    // overflow: '', //ILLEGAL!
```

Comment:
I had to do this be able to see the Issues screen on my simulator.
[overflow](https://reactnative.dev/docs/layout-props#overflow) only accepts one of these values `'visible', 'hidden', 'scroll'`.